docs/extend/plugins.md

```diff
@@ -1,87 +1,272 @@
 <!--[metadata]>
 +++
-title = "Extending Engine with plugins"
-description = "How to add additional functionality to Docker with plugins extensions"
-keywords = ["Examples, Usage, plugins, docker, documentation, user guide"]
+aliases = [
+"/engine/extend/"
+]
+title = "New Plugin System"
```

Comment:
SGTM.

Also I like the idea of using a more specific/descriptive folder name, but we can handle that in a later revision.